src/neo/SmartContract/InteropService.Contract.cs

```diff
@@ -134,6 +134,8 @@ private static bool Contract_CallEx(ApplicationEngine engine)
 
             private static bool Contract_CallEx(ApplicationEngine engine, UInt160 contractHash, string method, Array args, CallFlags flags)
             {
+                if (method.StartsWith('_')) return false;
```

Comment:
```suggestion
                if (method == "_initialize") return false;
```
I think that we should have a reserved words list

Reply:
We should reserve all method names starting with `_`.

Reply:
Then this should be forced by the compiler (and here).

Reply:
This will prevent the user call, but it could be called by the developer if he use `Pointers` I think that this is not a problem, but we should think about it.

Reply:
It's not a problem.